week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -14,45 +14,50 @@ example before trying the exercise.
 
 /* .replace() allows us to add something where we removed something*/
 let greeting = "Good Morning";
-greeting.replace('Morning', 'Evening'); // outputs Good Evening
-
+greeting.replace("Morning", "Evening"); // outputs Good Evening
 
 /* .substring() allows us to remove things from strings */
 let dessert = "ice cream and pancakes";
 
 let newdessert = dessert.substring(0, 9);
 
-console.log(newdessert); // returns ice cream
+// console.log(newdessert); // returns ice cream
 
 /* + + + + + + + */
 /* Now try the exercise */
 
-function safeLevels() {
-
+function safeLevels(arr) {
+  const numOnly = arr.map(function (p) {
+    return p.substring(0, 4);
+  });
+  const safeLevel = numOnly.find((n) => n < "23.5" && n > "19.5");
```

Comment:
while this is not wrong, do you need the quotation mark for the comparison 
n will get casted to number

Reply:
I compare those first as strings with % together, like "23.5%". It worked fine for this case but I realised that I need to separate "%" so it would be more validate comparison for different cases. But I forgot to remove quotation marks.